app-ios/tutanota/Sources/AppDelegate.swift

```diff
@@ -5,9 +5,8 @@ class AppDelegate : UIResponder,
                     UIApplicationDelegate,
                     UNUserNotificationCenterDelegate {
   var window: UIWindow?
-
+  
```

Comment:
Just wanted to pitch in quickly & say we should get a code formatter for Swift like [SwiftFormat](https://github.com/nicklockwood/SwiftFormat). If we do not have it already, a linter like [SwiftLint](https://github.com/realm/SwiftLint) would be good to.

Reply:
I have created an [issue here](https://github.com/tutao/tutanota/issues/6446).